app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_amount = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_amount += 1
+
+    if masks_amount:
+        return f"Friends should buy {masks_amount} masks"
+    return f"Friends can go to {cafe.name}"
```

Comment:
I guess Serhii was talking about descriptive messages for classes, which you didn't implement.